Issue 3168

charlottejmc opened the pull request:
I am preparing to add our new EN editors' profile pictures and bios to the project team page.

Closes #3168 

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Add the appropriate "Label"
- [x] If this PR closes an Issue, add the phrase `Closes #ISSUENUMBER` to your summary above
- [x] Ensure the status checks pass: if you have difficulty fixing build errors, please contact our Publishing Manager @anisa-hawes 
- [x] Check the Netlify Preview: navigate to netlify/ph-preview/deploy-preview and click 'details' (at right)
- [x] Assign at least one individual or team to "Reviewers"
  - ~[ ] if the text needs to be translated, please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines), then assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing editor in your PR.~

charlottejmc committed:
Update ph_authors.yml

Add new EN editors

charlottejmc committed:
Add portraits for new EN editors

charlottejmc committed:
Update ph_authors.yml

Adjust indenting

charlottejmc committed:
Add EN editor's portrait

anisa-hawes commented:
Thank you for helping with this, @charlottejmc! We just need to greyscale Laura's portrait!

charlottejmc committed:
Delete avatars/laura-chapot.png

Image was in colour

charlottejmc committed:
Upload greyscale portrait

anisa-hawes committed:
Update ph_authors.yml

- Correct spelling of Österreichische Akademie der Wissenschaften
- Add Twitter/X handles for Massimiliano and Augustín

anisa-hawes committed:
Update ph_authors.yml

Remove Agustín's profile (adding instead to #3187)

anisa-hawes committed:
Delete avatars/agustin-cosovschi.png

Adding to #3187 instead

anisa-hawes committed:
Update ph_authors.yml

- Add middle name, line 3055
- Correct ORCID, line 3078

anisa-hawes committed:
Merge branch 'gh-pages' into Issue-3168

anisa-hawes committed:
Update ph_authors.yml

space

anisa-hawes commented:
(Replaced by #3193)
